Tasks/validate.js

```diff
@@ -1,24 +1,33 @@
 // Validate person name
+"use strict";
 
-const isValidate = (T) => {
-  if (!T) return false;
-  if (T === '') return false;
-  if (typeof T !== 'string') return false;
-  if (T.length === 0) return false;
-  if (!T.includes(' ')) return false;
-  {
-    for (C of T) {
-      if (C === ' ') continue;
-      if (
-        C.toLowerCase().charCodeAt(0) >= 97 &&
-        C.toLowerCase().charCodeAt(0) <= 122
-      ) {
-      } else {
-        return false;
-      }
-    }
-    return true;
+const checkValidName = (T) => {
+  const exceptions = [
+    !T,
+    T === "",
+    typeof T !== "string",
+    T.length === 0,
+    !T.includes(" "),
+  ];
+
+  for (const exception of exceptions) {
+    if (exception) return false;
+  }
+
+  const FLOOR_CHAR_NUM = 97;
+  const CEIL_CHAR_NUM = 122;
+
+  for (const charOfName of T) {
+    if (charOfName === " ") continue;
+
+    if (
+      charOfName.toLowerCase().charCodeAt(0) < FLOOR_CHAR_NUM ||
+      charOfName.toLowerCase().charCodeAt(0) > CEIL_CHAR_NUM
+    )
+      return false;
```

Comment:
- Remove double execution of `char.toLowerCase().charCodeAt(0)` expression
- Do not need even if-statement
```suggestion
    const charCode = char.toLowerCase().charCodeAt(0);
    return charCodeAt < FLOOR_CHAR_NUM || charCodeAt > CEIL_CHAR_NUM;
```